SimPEG/MT/Utils/MT1Danalytic.py

```diff
@@ -0,0 +1,111 @@
+# Analytic solution of EM fields due to a plane wave
+
+import numpy as np, SimPEG as simpeg
+
+def getEHfields(m1d,sigma,freq,zd,scaleUD=True):
```

Comment:
Can we just add them as kwargs here? and give defaults, then the order is not so important.